apps/playground-api/src/PlaygroundBlock.ts

```diff
@@ -10,10 +9,16 @@ export class PlaygroundBlock {
   constructor (private readonly client: ApiClient) {
   }
 
-  @Interval(3000)
-  async generate (): Promise<void> {
-    await this.client.call('generatetoaddress', [1, PlaygroundSetup.address, 1], 'number')
+  async generateToAddress (): Promise<void> {
+    const { masternodes, blocks }: mining.MiningInfo = await this.client.mining.getMiningInfo()
+    const operator = masternodes[blocks % masternodes.length].operator
+    await this.client.call('generatetoaddress', [1, operator, 1], 'number')
     const count = await this.client.blockchain.getBlockCount()
     this.logger.log(`generated new block - height: ${count}`)
   }
+
+  @Interval(3000)
+  async generate (): Promise<void> {
+    await this.generateToAddress()
+  }
```

Comment:
```suggestion
  @Interval(3000)
  async generate (): Promise<void> {
    const { masternodes, blocks }: mining.MiningInfo = await this.client.mining.getMiningInfo()
    const operator = masternodes[blocks % masternodes.length].operator
    await this.client.call('generatetoaddress', [1, operator, 1], 'number')
    const count = await this.client.blockchain.getBlockCount()
    this.logger.log(`generated new block - height: ${count}`)
  }
```

Given that `generateToAddress` and `generate` are identical, we don't need another method? 

The decorator is a behavior applied to that method, so IMO should not warrant a different name either.